examples/basic-video-example.html

```diff
@@ -134,15 +134,15 @@
     <div class="input-container">
       <input
         id="config-url"
-        value="https://demov3.net955210.contentfabric.io/config"
+        value="https://main.net955305.contentfabric.io/config"
         placeholder="Config URL">
       <input
         id="private-key"
         value=""
         placeholder="Private Key">
       <input
         id="content-id"
-        value=""
+        value="hq__FRHfVK9ppcowDmB6s48cZtHFbJ5Qgduq126eLXJBfPbrmkATowwzz9mjzMfns5B5WpHwBfFYfG"
```

Comment:
Let's use this dedicated example video: hq__CcdV4wnCNq9wv6jXpYeCQ2GE4FLQBFtVSSSt2XKfBJMrH89DFDGsfkpWWvBy16QBGGYeF5mLGo